frontend/src/components/teacher/dashboard/AssessmentsSection.tsx

```diff
@@ -57,23 +57,14 @@ const AssessmentsSection = () => {
       )}
       {!!data?.length && !error && !loading && (
         <Tabs onChange={(index) => setCurrentTab(TEST_SESSION_STATUSES[index])}>
-          <TabList border="none" gap={8}>
+          <TabList>
```

Comment:
this tab code is now almost identical to `DisplayAssessmentsPage`. could we refactor that into a reusable component?